default.yml

```diff
@@ -29,26 +29,19 @@ AllCops:
   DisabledByDefault: false
   NewCops: disable # to avoid warnings. TODO: remove this setting and add new cops from the warning.
 
-# A bunch of these layout cops are enabled by default,
-# but for some reason 'DisableByDefault' is set to 'true' above.
```

Comment:
This comment is not valid anymore (based on line 29)

Reply:
I've removed these lines 

Do you want me to remove the rest of the comment too?

Reply:
If the comment does not make sense anymore and you're making changes around it, sure :)